examples/nsm_consul/dns/README.md

```diff
@@ -0,0 +1,157 @@
+## Setup DNS for two clusters
```

Comment:
Why do we need this?

Can we just re-use setup from istio?

Reply:
We can, I will update the readme.